src/Commands/Release/Release.php

```diff
@@ -279,4 +285,14 @@ public function getIncludeOtherChanges()
         // Default value
         return false;
     }
+
+    /**
+     * Whether to include upgrade-only changes.
+     *
+     * @return bool
+     */
+    public function getIncludeUpgradeOnly()
+    {
+        return $this->input->getOption('include-upgrade-only');
```

Comment:
```suggestion
        return $this->input->hasOption('include-upgrade-only');
```

(Boolean parameters need to be checked for presence, not value - when the parameter wasn't included, an exception was being thrown.)